tests/test_actions_db.py

```diff
@@ -10,6 +10,9 @@
 import pytest
 import sqlite3
 
+os.chdir(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..\
+/melissa/'))
```

Comment:
Is it possible not change the directory on the th test? Maybe put it on conftest? 

- http://doc.pytest.org/en/latest/fixture.html#sharing-a-fixture-across-tests-in-a-module-or-class-session
- http://stackoverflow.com/questions/34466027/in-py-test-what-is-the-use-of-conftest-py-files